lib/Doctrine/DBAL/Types/ConversionException.php

```diff
@@ -75,10 +80,11 @@ static public function conversionFailedFormat($value, $toType, $expectedFormat,
      * @param mixed    $value
      * @param string   $toType
      * @param string[] $possibleTypes
+     * @param \Exception|null $previous
      *
      * @return \Doctrine\DBAL\Types\ConversionException
      */
-    static public function conversionFailedInvalidType($value, $toType, array $possibleTypes)
+    static public function conversionFailedInvalidType($value, $toType, array $possibleTypes, \Exception $previous = null)
```

Comment:
Not sure there is a need for this one, I did that one for the sake of consistency